pkg/service/controllers/setup.go

```diff
@@ -3,14 +3,9 @@ package controllers
 import (
 	"context"
 
-	"github.com/rancher/norman/controller"
-	"github.com/rancher/types/config"
+	"github.com/rancher/rancher/pkg/types/config"
 )
 
-func Start(ctx context.Context, namespace string, apiContext *config.UserOnlyContext) error {
-	return controller.SyncThenStart(ctx, 1,
-		apiContext.Cluster.ClusterAuthTokens(namespace).Controller(),
-		apiContext.Cluster.ClusterUserAttributes(namespace).Controller(),
-		apiContext.Core.ConfigMaps(namespace).Controller(),
-	)
+func Start(ctx context.Context, apiContext *config.UserOnlyContext) error {
+	return apiContext.Start(ctx)
```

Comment:
This seems like a significant change unrelated to adding go module support. Can you explain what's going on here? Why are we getting rid of the `SyncThenStart`?

Reply:
`SyncThenStart` was removed from norman sometime after 2.4, and 2.4 couldn't be used without some difficulty (`Starter` interface was completely removed from the controllers that we are using in rancher/rancher and was required for `SyncThenStart`). The generated go.mod from the vendor.conf was invalid, so dependencies would have had to be selectively updated to find compatible versions which seemed like too much effort to not update.